Source/monster.cpp

```diff
@@ -348,7 +348,7 @@ void PlaceGroup(int mtype, int num, UniqueMonsterPack uniqueMonsterPack, int lea
 		int y1 = yp;
 
 		if (num + ActiveMonsterCount > totalmonsters) {
-			num = totalmonsters - ActiveMonsterCount;
+			num = static_cast<int>(totalmonsters - ActiveMonsterCount);

```

Comment:
`num`, `totalmonsters`, and `placed` can all be unsigned types

Reply:
Done